cordova-src/android/BranchDevice.java

```diff
@@ -83,9 +83,9 @@ private void getInstallData(JSONArray args, CallbackContext callbackContext) {
 		JSONObject installPost = new JSONObject();
 		try {
 			boolean debug = args.optBoolean(0);
-			Log.d("BranchDevice", "GetInstallData debug value is " + debug);
+			Log.d("BranchDevice", "getInstallData debug value is " + debug);
```

Comment:
since you've already touched this, would "in getInstallData() the value of debug is" be slightly more clear?